### DISCUSSION TITLE
WOW!!!! Just WOW!!! (Thank you!)

### DISCUSSION THREAD

**Discussion by ElijahLynn** [mfcc64/youtube-musical-spectrum#3] [General]
As someone who has been learning how to sing for the past year, I have tried many spectrum tools and have come across one that is really useful for singing with my own voice but isn't that great for multi-frequency visualization and therefore isn't that good for learning other songs but THIS extension really fills a void for learning other songs!!

I like how I can tell the voice from the instruments in this song by the vibrato visualization: https://www.youtube.com/watch?v=450p7goxZqg

https://user-images.githubusercontent.com/1504756/128948808-d66507f6-1504-46ba-b757-e7226e7ef776.mp4

**Comment by mfcc64** [comment]
Thank you so much.
